contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,973 @@
+        super.execute();
+
+        // Action 1: Set dust limits for sUSDe-GHO and USDC-GHO DEXes
+        action1();
+
+        // Action 2: Set dust limits for sUSDe-GHO/USDC-GHO T4 vault
+        action2();
+
+        // Action 3: Reduce allowance of fUSDC and fUSDT
+        action3();
+
+        // Action 4: Remove Borrow Rewards handlers as Vault Auth
+        action4();
+
+        // Action 5: Remove Team Multisig as Auth on Unlaunched Vaults
+        action5();
+
+        // Action 6: Pause Limits for Unlaunched USDe Collateral Vaults
+        action6();
+
+        // Action 7: Pause Limits for Bugged DEXes
+        action7();
+
+        // Action 8: Set Launch Limits for RLP-USDC DEX
+        action8();
+
+        // Action 9: Set Launch Limits for Gold Vaults and DEX
+        action9();
+
+        // Action 10: Update Borrow Limits for sUSDe/USDC
+        action10();
+
+        // Action 11: Update Range Percentages for USD0-USDC DEX
+        action11();
+
+        // Action 12: Deposit stETH from Treasury to Lite
+        action12();
+
+        // Action 13: Update limits for wstETH-ETH, weETH-ETH DEXes and vaults
+        action13();
+    }
+
+    function verifyProposal() public view override {}
+
+    function _PROPOSAL_ID() internal view override returns (uint256) {
+        return PROPOSAL_ID;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    // @notice Action 1: Set dust limits for sUSDe-GHO DEXes
+    function action1() internal isActionSkippable(1) {
+        {
+            address sUSDe_GHO_DEX = getDexAddress(33);
+            {
+                // sUSDe-GHO DEX
+                {
+                    // sUSDe-GHO Dex
+                    DexConfig memory DEX_sUSDe_GHO = DexConfig({
+                        dex: sUSDe_GHO_DEX,
+                        tokenA: sUSDe_ADDRESS,
+                        tokenB: GHO_ADDRESS,
+                        smartCollateral: true,
+                        smartDebt: false,
+                        baseWithdrawalLimitInUSD: 10_000, // $10k
+                        baseBorrowLimitInUSD: 0, // $0
+                        maxBorrowLimitInUSD: 0 // $0
+                    });
+                    setDexLimits(DEX_sUSDe_GHO); // Smart Collateral
+
+                    DEX_FACTORY.setDexAuth(sUSDe_GHO_DEX, TEAM_MULTISIG, true);
+                }
+            }
+        }
+    }
+
+    // @notice Action 2: Set dust limits for sUSDe-GHO/USDC-GHO T4 vault
+    function action2() internal isActionSkippable(2) {
+        {
+            //sUSDe/GHO : USDC/GHO
+            address sUSDe_GHO_DEX_ADDRESS = getDexAddress(33);
+            address USDC_GHO_DEX_ADDRESS = getDexAddress(4);
+            address sUSDe_GHO__USDC_GHO_VAULT_ADDRESS = getVaultAddress(125);
+
+            {
+                // Update sUSDe-GHO<>USDC-GHO vault borrow shares limit
+                IFluidAdminDex.UserBorrowConfig[]
```

Comment:
@bergben @xma8lu Why no supply limits?

Reply:
@thrilok209 based on the comment, we do it after team MS initializes the vault
https://github.com/Instadapp/inst-governance/pull/110/commits/19a5e022918dcc1b5da6f157a3e79670c144e161#r2073005290

Reply:
yes team MS must initialize the dex, before that setting any limits would revert